pdfminer/pdffont.py

```diff
@@ -488,6 +488,13 @@ def __init__(self, descriptor, widths, default_width=None):
         self.leading = num_value(descriptor.get('Leading', 0))
         self.bbox = list_value(descriptor.get('FontBBox', (0, 0, 0, 0)))
         self.hscale = self.vscale = .001
+
+        # PDF RM 9.8.1 specifies /Descent should always be a negative number.
+        # PScript5.dll seems to produce Descent with a positive number, but
+        # text analysis will be wrong if this is taken as correct. So force
+        # descent to negative.
+        if self.descent > 0:
```

Comment:
Could you group all the code for `self.descent`? 11 lines above the numeric value is parsed and the additional logic is here.